packages/handlersjs-http/.componentsignore

```diff
@@ -4,5 +4,6 @@
   "HandlerError",
   "HttpHandlerOperationSecurityType",
   "HttpHandlerResponse",
-  "Logger"
+  "Logger",
+  "SyncService"
```

Comment:
So, as discussed, this should not be ignored, since this normally prevents configurations from loading such a component directly.